src/components/landing/HeadArticle/HeadArticle.js

```diff
@@ -21,40 +29,41 @@ const imagesPreload = [
   image8,
 ];
 
-class HeadArticle extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      currentImage: image1,
-      currentImageCounter: 0,
-    };
-  }
-
-  async componentDidMount() {
-    this.interval = setInterval(() => {
-      this.setState((prevState) => ({
-        currentImageCounter:
-          (prevState.currentImageCounter + 1) % imagesPreload.length,
-        currentImage: imagesPreload[prevState.currentImageCounter],
-      }));
+const HeadArticle = () => {
+  const [currentImage, setCurrentImage] = useState(0);
+
+  useEffect(() => {
+    const imageChange = setInterval(() => {
+      setCurrentImage(current =>
+        current === imagesPreload.length - 1 ? 0 : current + 1,
+      );
     }, 5000);
-  }
-
-  render() {
-    const { currentImage } = this.state;
-    return (
-      <article
-        style={{
-          backgroundImage: `url(${currentImage})`,
-          backgroundPosition: "center",
-          backgroundSize: "cover",
-          backgroundRepeat: "no-repeat",
-        }}
-      >
-        <HeadArticleSearch />
-      </article>
-    );
-  }
-}
+
+    return () => {
+      clearInterval(imageChange);
+    };
+  }, []);
+
+  return (
+    <article className="banner">
+      <div className="banner-content">
+        <div className="banner-text">
+          <h3>{BANNER_TEXT_1}</h3>
+          <h4>{BANNER_TEXT_2}</h4>
+        </div>
+        <Link to="/home/events" className="banner-button" type="button">
+          {BANNER_BUTTON_TEXT}
+        </Link>
+      </div>
+      <div className="banner-img">
```

Comment:
Extra `div`

Reply:
Which are? They all are used to place components